### DISCUSSION TITLE
Discord music bot and YouTube ToS

### DISCUSSION THREAD

**Discussion by wolf-yuan-6115** [eritislami/evobot#833] [General]
# Music bot is against YouTube ToS

Recently, Groovy, a huge discord music bot received Google alert. Google said Groovy bot is against YouTube ToS and they need to shutdown groovy in 7 days.
Is Discord music bot against YouTube ToS?
Actually, **yes**, YouTube is not allowed to download videos from it, even is on Discord bot.

In future, Google might will content Discord and let music bot against ToS, and all bot will be deleted or owner will got banned.

YouTube Terms of service
> The following restrictions apply to your use of the Service. You are not allowed to:
>
> access, reproduce, download, distribute, transmit, broadcast, display, sell, license, alter, modify or otherwise use any part of the Service or any Content except: (a) as expressly authorized by the Service; or (b) with prior written permission from YouTube and, if applicable, the respective rights holders;

> See also:
> https://www.theverge.com/2021/8/24/22640024/youtube-discord-groovy-music-bot-closure
>
> https://www.pcgamer.com/google-has-killed-discords-best-music-bot/

**Comment by SrIzan10** [comment]
So that means I'm going to get banned if I continue hosting a Discord Bot?

**Comment by eritislami** [comment]
Gonna convert this to discussion

**Comment by nullptr448** [comment]
This will only affect big bots tho (Like Rythm, Fredboat, etc..) If you are for example hosting your own music bot I think you don't have anything to worry about.

**Reply by SrIzan10** [reply]
Yeah, but I'm going to shutdown the bot anyways. Maybe Google could see my API requests and block me or something. You know: Google is Google.

**Reply by nullptr448** [reply]
Yeah, I think these measures are a bit exaggerated but the rules are the rules, And you certainly don't want to get in legal trouble with Google.  `¯\_(ツ)_/¯`

**Reply by SrIzan10** [reply]
And I don't want to (probably) lose my Google and Discord account.

**Reply by wolf-yuan-6115** [reply]
idk what will happened next ...
